api/pyproject.toml

```diff
@@ -190,6 +190,7 @@ zhipuai = "1.0.7"
 # Related transparent dependencies with pinned version
 # required by main implementations
 ############################################################
+pymochow = "1.3.1"
```

Comment:
Avoid repeating the dependency crosssing groups. Moreover, it's used for vdb clients, which should not be put in main or indirect dependency group.

Reply:
thanks, fixed

Reply:
LGTM.